.github/workflows/deploy-release.yml

```diff
@@ -202,13 +206,15 @@ jobs:
           
           cp Package.swift.template Package.swift
           shasum -a 256 lipalightninglibFFI.xcframework.zip | sed 's/ .*//' > checksum
-          sed -i "s/to_replace_release_version/${{ env.RELEASE_VERSION }}/g" Package.swift
+          sed -i "s/to_replace_release_version/${{ env.RELEASE_VERSION }}${{ matrix.mock-deps && '-mock' || '' }}/g" Package.swift
           sed -i "s/to_replace_zip_checksum/$(cat checksum)/g" Package.swift
           
           git add Package.swift
           git add Sources/LipaLightningLib/LipaLightningLib.swift
           git commit -m "This commit was created automatically by the lipa bot"
-          git tag -a ${{ env.RELEASE_VERSION }} HEAD -m "This tag was created automatically by the lipa bot."
+          git restore .
```

Comment:
Why is this restore needed?

Reply:
When we copy around files to create `lipalightninglibFFI.xcframework.zip` before, that creates unstaged changes. With unstaged changes, we cannot `git pull --rebase`. So `git restore .` gets rid of all unstaged changes.

The other question though, is whether `git pull --rebase` is even still needed after I made the to deployments (regular and mock) [run sequentially](https://github.com/getlipa/lipa-lightning-lib/pull/934/commits/645bf9e24aa9018d35cf55fff110d226d472e050). Probably not 🤷
But also it doesn't hurt anyone.